include/merlin/core_kernels.cuh

```diff
@@ -1448,44 +1448,61 @@ __global__ void remove_kernel(const Table<K, V, M>* __restrict table,
 }
 
 /* Dump with meta. */
+template <class K, class V, class M>
+inline std::tuple<size_t, size_t> dump_kernel_shared_memory_size(
+    const size_t available_shared_memory) {
+  struct KVM {
```

Comment:
Meta struct definition is not stable, it will be extended in the future, so we'd better move the `KVM` to `types.cuh` and close to the `Meta` definition.

Reply:
Done!